platform-sdk/swirlds-fchashmap/src/main/java/com/swirlds/fchashmap/config/FCHashMapConfig.java

```diff
@@ -29,9 +29,17 @@
  * 		the amount of time that must pass between error logs about the FCHashMapGarbageCollector's queue size.
  * @param archiveEnabled
  * 		is the archival of FCHashMap enabled?
+ * @param rebuildSplitFactor
+ *      split the binary tree at this depth relative to the root of the binary tree. The tree will be split into 2^split-factor subtrees, and each subtree will be eligible to be handled on a separate thread.
+ * @param rebuildThreadCount
+ *      rebuilding the FCHashMap in a MerkleMap, use this many threads to rebuild the tree.
+ *
  */
 @ConfigData("fcHashMap")
 public record FCHashMapConfig(
         @ConfigProperty(defaultValue = "200") int maximumGCQueueSize,
         @ConfigProperty(defaultValue = "1m") Duration gCQueueThresholdPeriod,
-        @ConfigProperty(defaultValue = "true") boolean archiveEnabled) {}
+        @ConfigProperty(defaultValue = "true") boolean archiveEnabled,
+        @ConfigProperty(defaultValue = "7") int rebuildSplitFactor,
+        // TODO: Runtime.getRuntime().availableProcessors()
+        @ConfigProperty(defaultValue = "4") int rebuildThreadCount) {}
```

Comment:
We should fix this before the PR can be merged. We can create a custom ConfigSource to calc the value